app/src/main/java/com/keylesspalace/tusky/adapter/StatusDetailedViewHolder.java

```diff
@@ -52,13 +55,18 @@ protected void setCreatedAt(Date createdAt, StatusDisplayOptions statusDisplayOp
 
     private void setReblogAndFavCount(int reblogCount, int favCount, StatusActionListener listener) {
 
-        if (reblogCount > 0) {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(itemView.getContext());
+
+        // If wellbeing mode is enabled, favs and boosts should not be visible.
+        boolean wellbeingEnabled = preferences.getBoolean(PrefKeys.WELLBEING_HIDE_STATS_POSTS, false);
```

Comment:
same here, better load the setting somewhere else and pass the value to the Adapter (maybe put it in the `StatusDisplayOptions`)